CHANGELOG.md

```diff
@@ -27,7 +27,7 @@ This project adheres to [Semantic Versioning].
 - [#1625] Fix the wrong CSRF token storage being wired ([Toflar])
 - [#1628] Adjust the FrontendController::checkCookiesAction() comment ([Mynyx])
 - [#1638] Correctly check if the search panel is active ([dmolineus])
-- [#1658] Add the Contao Manager cookie to the cookie blacklist ([Mynyx])
+- [#1658] Add the Contao Manager cookie to the cookie disallow-list ([Mynyx])
```

Comment:
This will be overwritten by my changelog generator script again. We need to adjust the issue title on github.com to make it a permanent change.

Reply:
Ok, I'll revert that then. Can you adjust the issues?

Reply:
Done. They both use the term "cookie deny list" now, so if we should end up with something other than "deny list", please remind me to adjust the issue titles again. 😄